Docs/push-credentials-via-notify-api.md

```diff
@@ -0,0 +1,29 @@
+## Create Push Credentials via the Notify Credential Resource API
+
+Voice SDK users can manage their Push Credentials in the developer console (**Console > Account > Keys & Credentials > Credentials**). Currently the Push Credential management page only supports the default region (US1). To create or update Push Credentials for AU1 regional usage, developers can use the AU1 endpoint of the Notify public API to manage their Push Credentials. Follow the instructions of the [Credential Resource API](https://www.twilio.com/docs/notify/api/credential-resource) and replace the endpoint with the AU1 API endpoint (https://notify.sydney.au1.twilio.com).
```

Comment:
Should we  generalize this `AU1 regional usage` .. so that when we GA the other regions, these documentations still apply?
s/`To create or update Push Credentials for AU1 regional usage, ...` / `To create or update Push Credentials for other regional (i.e. AU1) usage, ....`